scripts/camera/follow-camera.js

```diff
@@ -17,7 +17,7 @@ FollowCamera.attributes.add('lerpAmount', {
     max: 1,
     default: 0.99,
     title: 'Lerp Amount',
-    description: 'The amount to lerp the camera towards its desired position every frame based on a 60fps frame rate. Lerping is frame rate independent though and will be correct for other frame rates.'
+    description: 'The amount to lerp the camera towards its desired position over time. Adjust the value until it feels right for the app. Lerping is frame rate independent though and will be correct for other frame rates.'
```

Comment:
personally, the original text feels more like something you can work with and understand, than the new text?

Reply:
The old text is no longer correct. The lerp amount is ultimately a 'fudge' number that is changed until it 'feels' right.

Reply:
yeah I like that .. gives them some useful info

Reply:
```suggestion
    description: 'The amount to lerp the camera towards its desired position over time. The closer it is to 1, the faster the camera will move. Lerping is frame rate independent though and will be correct for other frame rates.'
```
```suggestion
    description: 'The amount to lerp the camera towards its desired position over time. Adjust the value until it feels right for the app. Lerping is frame rate independent though and will be correct for other frame rates.'
```